zuul.d/project.yaml

```diff
@@ -125,6 +125,7 @@
         - ansible-security-trendmicro-deepsec-appliance:
             files:
               - playbooks/ansible-network-appliance-base/.*
+              - roles/tm.*
```

Comment:
You can remove the files section for your test. This way `ansible-security-tredmicro-deepsec-appliance` will be always executed, even if there is no changes in these directories